packages/nx-monorepo/src/nx-monorepo.ts

```diff
@@ -239,6 +305,92 @@ export class NxMonorepoProject extends TypeScriptProject {
     });
   }
 
+  /**
+   * Helper to format `npx nx run-many ...` style command.
+   * @param options
+   */
+  public formatNxRunManyCommand(options: NxRunManyOptions): string {
+    const cmd: string[] = [
+      "npx nx run-many",
+      `--target=${options.target}`,
+      `--output-style=${options.outputStyle || "stream"}`,
+    ];
+    if (options.configuration) {
```

Comment:
These options can be simplified to `options.configuration && cmd.push(...)`

Reply:
True - but the lines will break regardless and likely be less readable